From 9e5820a86277818c2f83c11c2aa76d7f0a38283e Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <vda.linux@googlemail.com>
Date: Sun, 14 Aug 2016 02:54:27 +0200
Subject: build system: fix include/NUM_APPLETS.h generation

TBH, it's more like "work around my bad makefile-fu" than "fix"...

Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>
---
 applets/Kbuild.src | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

Upstream-Status: Backport

diff --git a/applets/Kbuild.src b/applets/Kbuild.src
index b612399..5cc1827 100644
--- a/applets/Kbuild.src
+++ b/applets/Kbuild.src
@@ -29,7 +29,7 @@ applets/applets.o: include/usage_compressed.h include/applet_tables.h
 
 applets/applet_tables: .config include/applets.h
 applets/usage:         .config include/applets.h
-applets/usage_pod:     .config include/applet_tables.h include/applets.h
+applets/usage_pod:     .config include/applets.h include/applet_tables.h include/NUM_APPLETS.h
 
 quiet_cmd_gen_usage_compressed = GEN     include/usage_compressed.h
       cmd_gen_usage_compressed = $(srctree_slash)applets/usage_compressed include/usage_compressed.h applets
@@ -40,8 +40,5 @@ include/usage_compressed.h: applets/usage $(srctree_slash)applets/usage_compress
 quiet_cmd_gen_applet_tables = GEN     include/applet_tables.h
       cmd_gen_applet_tables = applets/applet_tables include/applet_tables.h include/NUM_APPLETS.h
 
-include/applet_tables.h: applets/applet_tables
-	$(call cmd,gen_applet_tables)
-
-include/NUM_APPLETS.h: applets/applet_tables
+include/applet_tables.h include/NUM_APPLETS.h: applets/applet_tables
 	$(call cmd,gen_applet_tables)
-- 
cgit v0.12

